### PR TITLE
Add a rule about trailing commas in lists/maps/structs/tuples

### PR DESCRIPTION
**Pull request by whatyouhide**
It's about time! :dancers:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,26 @@
...
+    :foo,
+    :bar,
+    :baz
+  ]
+
+  # Good
+  [
+    :foo,
+    :bar,
+    :baz,
```
Comment (b0oh) [inline README.md:128]: WHAT WHAT WHAT?????
I am strongly against that.

Comment (whatyouhide) [inline README.md:128]: I'm enough in favour of that to convince every one, I'm not scared :) Easier reordering of items, less noisy diff when adding/removing items, really really weak arguments in favour of it. What's yours? :stuck_out_tongue:

**Comment by whatyouhide** [comment]
@lexmag updated to say "advised".

**Comment by b0oh** [comment]
So if it is "advised" then maybe instead of "Bad" and "Good" use something different? :)

**Comment by lexmag** [comment]
@b0oh good point, probably just show only how it looks with trailing comma.

**Comment by whatyouhide** [comment]
@b0oh @lexmag I thought about this and yeah, better. I went with "Preferred" and showed only the case with the trailing comma. :)

**Comment by whatyouhide** [comment]
🎉

**Comment by lexmag** [comment]
💛

**Pull request merged**